tensorboard/webapp/metrics/views/card_renderer/scalar_card_types.ts

```diff
@@ -56,6 +66,11 @@ export interface ScalarCardPoint extends Point {
 
 export type ScalarCardDataSeries = DataSeries<ScalarCardPoint>;
 
+export interface RunStepData {
+  metadata: ScalarCardSeriesMetadata;
+  closestStartPoint: ScalarCardPoint;
```

Comment:
Can we then make this to 
```
{
displayName: string,
color: .. // do we need color?
smoothed: number, // not sure where to get this
value: number,
step: number,
}
```
The idea of "giving a bunch of data and show what headers have" looks okay to me, but it is just confusing to me why we need to have two layers of that data (i.e., separate them to metadata and closestStartPoint)

Also if we want to show min/max later, do you plan to calculate this and passed to the component or do thin inside the component?
If the former, we need to extend this interface (to have min/max etc); if the latter, we then need to passed in all the data points.
I might prefer to do this as in the first option considering recalculating on updating the range.